vocabularies/Analytics.json

```diff
@@ -211,6 +211,13 @@
             "$Type": "Analytics.AutoExpandEntry",
             "@Core.Description": "Entries with exceptional expansion",
             "@Core.OptionalParameter": {}
+          },
+          {
+            "$Name": "SubtotalsAtBottom",
+            "$Type": "Edm.Bool",
+            "@Core.Description": "Whether to have expanded subtotal rows duplicated: before and after their descendants",
```

Comment:
Whether to duplicate the group headers so that they are included before and after their descendants